pkg/pods/pod.go

```diff
@@ -229,11 +229,12 @@ func (pod *Pod) buildRunitServices(launchables []launch.Launchable, restartPolic
 				return util.Errorf("Duplicate executable %q for launchable %q", executable.Service.Name, launchable.ID())
 			}
 			sbTemplate[executable.Service.Name] = runit.ServiceTemplate{
+				Log: newManifest.GetLogExecutable(),
```

Comment:
So maybe the answer you're looking for is 1) preserve the `log_exec` section of the pod manifest 2) also add a config section for the preparer to supply a `default_log_exec` (see `pkg/preparer/setup.go`) 3) use the pod's log exec as an override to the preparer's default.